lambdas/db-migration/src/migrations/20201014105030_create_async_operations_table.ts

```diff
@@ -14,7 +14,7 @@ export const up = async (knex: Knex): Promise<void> =>
       .comment('description for async operation')
       .notNullable();
     table
-      .enum('operationType', ['ES Index', 'Bulk Granules', 'Bulk Granule Delete', 'Kinesis Replay', 'Reconciliation Report'])
+      .enum('operation_type', ['ES Index', 'Bulk Granules', 'Bulk Granule Delete', 'Kinesis Replay', 'Reconciliation Report'])
```

Comment:
I think you need to update the code in the new `async-operations` package that writes to RDS to match these new column names